libs/journeys/ui/src/libs/EditorProvider/EditorProvider.tsx

```diff
@@ -85,6 +86,7 @@ export interface EditorState {
    */
   selectedStep?: TreeBlock<StepBlock>
   steps?: Array<TreeBlock<StepBlock>>
+  journeyStatsBreakdown: StatsBreakdownFields[]
```

Comment:
Lets make this nullable so we don't have to update other components/tests 
```suggestion
  journeyStatsBreakdown?: StatsBreakdownFields[]
```